### PR TITLE
Update ui-template and sync UI

### PR DESCRIPTION
**Pull request by wankai123**
- Update ui-template, fix some dashboards,s and add dashboard brief descriptions.
- Sync UI to: https://github.com/apache/skywalking-booster-ui/commit/8d87777832620721b33ba5c46067fefe0d50efa9

### CODE REVIEW

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/browser/browser-root.json [hunk 1]
```diff
@@ -51,12 +51,35 @@
...
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
+            "backgroundColor": "white",
+            "content": "Provides Browser-Side monitoring of Web-App, Versions and Pages, through Apache SkyWalking Client JS.",
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/browser/browser-root.json:71]:
```suggestion
            "content": "Provide Browser-Side monitoring of Web-App, Versions and Pages, through Apache SkyWalking Client JS.",
```

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-root.json [hunk 2]
```diff
@@ -38,14 +38,36 @@
...
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
+            "backgroundColor": "white",
+            "content": "Provides monitoring of the status and resources of the K8S Cluster.",
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-root.json:65]:
```suggestion
            "content": "Provide monitoring of the status and resources of the K8S Cluster.",
```

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s_service/k8s-service-root.json [hunk 3]
```diff
@@ -42,6 +42,29 @@
...
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
+            "backgroundColor": "white",
+            "content": "Observe Service status and resources from K8S side.",
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s_service/k8s-service-root.json:62]:
```suggestion
            "content": "Observe Service status and resources from Kubernetes.",
```

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/mesh_cp/mesh-control-plane-root.json [hunk 4]
```diff
@@ -38,14 +38,36 @@
...
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
+            "backgroundColor": "white",
+            "content": "Provides monitoring of the behavior of Istio through its self-monitoring metrics.",
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/mesh_cp/mesh-control-plane-root.json:65]:
```suggestion
            "content": "Provide monitoring of the behavior of Istio through its self-monitoring metrics.",
```

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-root.json [hunk 5]
```diff
@@ -51,6 +51,29 @@
...
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
+            "backgroundColor": "white",
+            "content": "Provides monitoring of the status and resources of the VMs running Linux.",
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-root.json:71]:
```suggestion
            "content": "Provide Linux OS monitoring.",
```

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-root.json [hunk 6]
```diff
@@ -51,6 +51,29 @@
...
+          "i": "100",
+          "type": "Text",
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-root.json:69]: Are we changing all links to blue? Some were black only.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/virtual_database/virtual-database-root.json [hunk 7]
```diff
@@ -51,6 +51,29 @@
...
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
+            "backgroundColor": "white",
+            "content": "Observe the Virtual-Database which is conjectured by client side agent plugin.",
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/virtual_database/virtual-database-root.json:71]:
```suggestion
            "content": "Observe the Virtual Database which is conjectured by language agent through various plugins.",
```

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-root.json (hunk 6 above)
Comment (wankai123) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-root.json:69]: I changed all links to blue, then users could notice it's a link.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**